platform/ui/src/components/ToolbarButton/ToolbarButton.tsx

```diff
@@ -31,58 +28,78 @@ const ToolbarButton = ({
   );
 
   const sizeToUse = size ?? 'toolbar';
-  const toolTipClassNameToUse =
```

Comment:
why did we remove this? we need it to size the buttons dynamically for the toolbox in the segmentation panel vs the buttons in the toolbar, if you take a look now you will notice that its broken